presto-product-tests/README.md

```diff
@@ -176,6 +176,15 @@ where profile is one of either:
  `mysql_connector` and `postgresql_connector` tests i.e.
  `-x mysql_connector, postgresql_connector`.
 
+### Hadoop docker image used for testing
+The default Hadoop/Hive docker image used for testing is defined in `conf/common/compose-commons.sh` and can be controlled via the `HADOOP_BASE_IMAGE` and `DOCKER_IMAGES_VERSION` env variables.
+- `HADOOP_BASE_IMAGE` defines the Hadoop distribution family (as found in [PrestoDB Hadoop docker repo](https://cloud.docker.com/swarm/prestodb/repository/list?name=hive&namespace=prestodb)).
+The name should be without the `-kerberized` postfix, eg. `cdh5.13-hive`. Images that have their kerberized counterpart can be used in with test profiles implying a kerberized environment
```

Comment:
`Images that have their kerberized counterpart can be used in with test profiles implying a kerberized environment...`
This sentence doesn't parse. Could you rephase what do you mean?

Reply:
done